src/modules/micrortps_bridge/CMakeLists.txt

```diff
@@ -106,7 +106,6 @@ if (GENERATE_RTPS_BRIDGE)
 			-e templates/uorb_microcdr
 			-t ${CMAKE_CURRENT_BINARY_DIR}/tmp/headers_microcdr
 			-q
-		DEPENDS ${msg_files}
```

Comment:
We need a dependency here so that it's regenerated.

Reply:
Before this commit ${msg_files} is evaluated to empty, as $(msg_files) is a local variable in msg/CMakeList.

Reply:
Any ideas how to fix it? Not having dependencies isn't okay because developers will make changes and end up in a state that doesn't make any sense until they do a clean build.

Reply:
I am not very clear about how micrortps is organized, did not have enough time to look into it. 
Looks to me like some bug here, when this command builds OK with an empty dependency, but fail to build with some real value in ${msg_files}.

So what is the original intention to put a dependency here? is it really targeting on make dependency on msg_files, or this variable is just randomly selected as a workaround?

Reply:
If you touch a .msg file then these steps need to rerun. Not having a dependency here is even more evil than glob.